samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -115,16 +119,30 @@ CamResolutionsTest.prototype = {
     var avgEncodeMs = arrayAverage(googAvgEncodeTime);
     var minEncodeMs = arrayMin(googAvgEncodeTime);
     var maxEncodeMs = arrayMax(googAvgEncodeTime);
-    report.traceEventInstant('encode-stats', { width: currentRes[0],
+    var avgFPSInput = arrayAverage(googAvgFrameRateInput);
+    var minFPSInput = arrayMin(googAvgFrameRateInput);
+    var maxFPSInput = arrayMax(googAvgFrameRateInput);
+    var avgFPSSent = arrayAverage(googAvgFrameRateSent);
+    var minFPSSent = arrayMin(googAvgFrameRateSent);
+    var maxFPSSent = arrayMax(googAvgFrameRateSent);
+    report.traceEventInstant('video-stats', { width: currentRes[0],
                                                height: currentRes[1],
                                                minEncodeMs: minEncodeMs,
                                                maxEncodeMs: maxEncodeMs,
-                                               avgEncodeMs: avgEncodeMs });
+                                               avgEncodeMs: avgEncodeMs,
+                                               minFPSInput: minFPSInput,
+                                               maxFPSInput: maxFPSInput,
+                                               avgFPSInput: avgFPSInput,
+                                               minFPSSent: minFPSSent,
+                                               maxFPSSent: maxFPSSent,
+                                               avgFPSSent: avgFPSSent });
 
     if (googAvgEncodeTime.length === 0) {
       reportError('No stats collected. Check your camera.');
     } else {
       reportInfo('Encode time (ms): ' + minEncodeMs + ' min / ' + avgEncodeMs + ' avg / ' + maxEncodeMs + ' max');
+      reportInfo('Input FPS: ' + minFPSInput + ' min / ' + avgFPSInput + ' avg / ' + maxFPSInput + ' max');
+      reportInfo('Sent FPS: ' + minFPSSent + ' min / ' + avgFPSSent + ' avg / ' + maxFPSSent + ' max');
```

Comment:
do you wanna add a check here check for:
 if (avgFPSSent < 5) { reportError('Very low framerate.'); } ?